src/backends/perf_pt/collect.c

```diff
@@ -187,7 +187,7 @@ handle_sample(void *aux_buf, struct perf_event_mmap_page *hdr,
     //    re-use the space we have just "marked free" before we copied it.
     //
```

Comment:
Now you've got two blank lines?

Reply:
Eh? The comment in full is:
```
      // We need to use atmoics with orderings to protect against 2 cases.    
      //    
      // 1) It must not be possible to read the data buffer before the most    
      //    recent head is obtained. This would mean that we may read nothing when
      //    there is really data available.    
      //    
      // 2) We must ensure that we have already copied out of the data buffer    
      //    before we update the tail. Failure to do so would allow the kernel to 
      //    re-use the space we have just "marked free" before we copied it.    
      //    
      // The initial load of the tail is relaxed since we are the only thread        
      // mutating it and we don't mind variations on the ordering.    
      //    
      // See the following comment in the Linux kernel sources for more:    
      // https://github.com/torvalds/linux/blob/3be4aaf4e2d3eb95cce7835e8df797ae65ae5ac1/kernel/events/ring_buffer.c#L60-L85
```

Am I being silly?

Reply:
Nope, I seemed to have misread the diff. Sorry.